crates/test_utils/src/main.rs

```diff
@@ -97,11 +96,11 @@ fn main() {
             }
             // Handle other ConnectorAuthType variants
             _ => {
-                eprintln!("Invalid authentication type.");
+                eprintln!("\x1b[31mInvalid authentication type.\x1b[0m");
```

Comment:
Any reason for adding the ansi codes. these are OK for terminal but not useful for github actions.

Reply:
This did work in GitHub actions when I tested it yesterday. I'll remove this for `--color` command in Newman